analyzer/src/funTest/kotlin/managers/NuGetFunTest.kt

```diff
@@ -20,14 +20,10 @@
 package org.ossreviewtoolkit.analyzer.managers
 
 import io.kotest.core.spec.style.StringSpec
-import io.kotest.matchers.collections.containExactly
-import io.kotest.matchers.should
 import io.kotest.matchers.shouldBe
 
 import java.io.File
 
-import org.ossreviewtoolkit.analyzer.managers.utils.NuGetDependency
-import org.ossreviewtoolkit.analyzer.managers.utils.OPTION_DIRECT_DEPENDENCIES_ONLY
```

Comment:
With this removal, the `OPTION_DIRECT_DEPENDENCIES_ONLY` constant becomes unused. Please remove it from `NuGetSupport.kt`.

Reply:
Done!